CMakeLists.txt

```diff
@@ -403,6 +403,18 @@ else ()
     message("Skipping g2.dat generation in macOS cross-compile")
 endif ()
 
+# font
+if (NOT (MACOS_BUNDLE AND (NOT CMAKE_OSX_ARCHITECTURES MATCHES "${SYSTEM_MACOS_ARCH}")))
+    add_custom_command(
+            OUTPUT font.dat
+            COMMAND ./openrct2-cli sprite build ${CMAKE_BINARY_DIR}/font.dat ${ROOT_DIR}/resources/font/sprites.json
+            WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
+    )
+    add_custom_target(font DEPENDS OpenRCT2::openrct2-cli font.dat)
+else ()
+    message("Skipping font.dat generation in macOS cross-compile")
+endif ()
+
```

Comment:
Ideally we would write a function that could tackle both g2 and font(s).dat, rather than duplicating the block. Not sure how best to do that in CMake syntax, though. @duncanspumpkin, any suggestions?

(This can wait; it's not a blocker for merger, for me.)